tests/v1/core/test_scheduler.py

```diff
@@ -242,7 +242,9 @@ def test_schedule_partial_requests():
     model_runner_output = ModelRunnerOutput(
         req_ids=[request.request_id for request in requests],
         req_id_to_index=req_to_index,
-        sampled_token_ids=[[0] for _ in range(len(requests))],
+        # Only the first request has a sampled token id because
+        # the rest requests are still being prefilled.
+        sampled_token_ids=[[0], [], []],
```

Comment:
A dumb question: Is this change relevant to this PR? Or is it just a bug in this test?

Reply:
Yes this is relevant. Before this PR all requests would have `sampled_token_ids`. With this PR, the rest 2 requests being prefilled won't have `sampled_token_ids` (so empty list).

Reply:
thanks for the explanation.